css-scroll-snap-2/Overview.bs

```diff
@@ -193,26 +193,117 @@ The 'scroll-start' property {#scroll-start}
+					property along the block axis.
+			1. Return the nearest <a>scroll offset</a> that aligns |scrollcontainer| with
+				|target| along the block axis according to |alignment|.
+		1. Otherwise:
+			1. If the value of |target|'s [[css-scroll-snap-1#scroll-snap-align|scroll-snap-align]]
+				property along the inline axis is not ''none'':
+					1. Let |alignment| be the value of |target|'s [[css-scroll-snap-1#scroll-snap-align|scroll-snap-align]]
+						property along the inline axis.
+			1. Return the nearest <a>scroll offset</a> that aligns |scrollcontainer| with
+				|target| along inline axis according to |alignment|.
+
+	<h4 id="scroll-start-target-longhands">Longhands</h4>
+
+		<pre class="propdef">
+			Name: scroll-start-target-block, scroll-start-target-inline, scroll-start-target-x, scroll-start-target-y
+			Value: auto | none
+			Initial: none
+			Applies to: all elements
+			Inherited: no
+			Logical property group: scroll-start-target
+			Percentages: n/a
+			Computed Value: either of the keywords "none" or "auto"
+			Animation type: not animatable
+		</pre>
+
+		{{scroll-start-target-block}} and {{scroll-start-target-inline}} specify the
+		{{scroll-start-target}} values for the block and inline axis respectively.
+		{{scroll-start-target-y}} and {{scroll-start-target-x}} specify the
+		{{scroll-start-target}} values for the vertical and horizontal axis respectively.
+
+		<dl dfn-type=value dfn-for="scroll-start-target, scroll-start-target-x, scroll-start-target-y, scroll-start-target-block, scroll-start-target-inline">
+			<dt><dfn>none</dfn>
+			<dd>Element is not a ''scroll-start-target'' in the relevant axis.
+			<dt><dfn>auto</dfn>
+			<dd>Element is a ''scroll-start-target'' and is used to determine the
+				<a>initial scroll position</a> of its nearest ancestor <a>scroll container</a>
+		</dl>
+
+	<h4 id="scroll-start-target-shorthand">The scroll-start-target shorthand</h4>
+
+		{{scroll-start-target}} is a shorthand property that sets all of the scroll-start-target-*
+		longhands in one declaration. The first value defines the <a>initial scroll  position</a>
+		in the block axis, and the second sets it in the inline axis. If the second value
+		is omitted, it is the same as the specified first value.
+
+		<pre class="propdef shorthand">
+			Name: scroll-start-target
+			Value: [ none | auto ]{1,2}
+			Initial: ''none''
+			Applies to: all elements
+			Inherited: no
+			Percentages: N/A
+			Computed Value: see individual properties
+			Animation type: none
+		</pre>
+
+		Values are for the {{scroll-start-target}} shorthand have the same definitions
+		as values for the [[#scroll-start-target-longhands|longhands]].
+
+	<h4 id="scroll-start-target-with-place-content">Interaction with <a>place-content</a></h4>
+
+	If the <a>initial scroll position</a> indicated by <a spec="css-align" lt="place-content">place-content</a> differs
+	from that indicated by {{scroll-start-target}}, user-agents should give preference
+	to the latter.
```

Comment:
Shouldn't this take precedence over place-content? It seems like `scroll-start-target` is a more precise location whereas place-content can only determine whether scrolling should start at the beginning or end of a scroller.

Reply:
I think that's what I'm saying? the former would be the "initial scroll position indicated by place-content" and the latter would be the initial scroll position "indicated by scroll-start-target" and user agents should give preference to the latter.